libraries/AP_BattMonitor/AP_BattMonitor_Params.cpp

```diff
@@ -162,6 +162,14 @@ const AP_Param::GroupInfo AP_BattMonitor_Params::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("ARM_MAH", 19, AP_BattMonitor_Params, _arming_minimum_capacity, 0),
 
+    // @Param: BUS
+    // @DisplayName: Battery monitor I2C bus number
+    // @Description: Battery monitor I2C bus number
+    // @Range: 0 5
```

Comment:
Pretty sure the highest value we have for a range on this is actually 2 or maybe 3? @tridge

Reply:
It's apparently 3 for the Durandal, fixed, thanks!